### PR TITLE
[KotlinCleanup] OverviewStatsBuilder.kt

### PR DESCRIPTION
**Pull request by Jatin887**
## Purpose / Description
Kotlin CleanUp

## Fixes
Part of [https://github.com/ankidroid/Anki-Android/issues/10489]

## Approach
Using lint and some code changes which were mentioned in the comments and required to make the code more reliable

## How Has This Been Tested?
Physical Device One Plus Nord2

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/stats/OverviewStatsBuilder.kt [hunk 1]
```diff
@@ -126,110 +122,109 @@ class OverviewStatsBuilder(private val webView: WebView, private val col: Collec
...
-        @KotlinCleanup("com.ichi2.R.string... => R.string...")
-        stringBuilder.append(res.getString(com.ichi2.anki.R.string.stats_overview_forecast_total, oStats.forecastTotalReviews))
+        stringBuilder.append(_subtitle(res.getString(string.stats_forecast).uppercase(l)))
+        stringBuilder.append(res.getString(string.stats_overview_forecast_total, oStats.forecastTotalReviews))
         stringBuilder.append("<br>")
-        stringBuilder.append(res.getString(com.ichi2.anki.R.string.stats_overview_forecast_average, oStats.forecastAverageReviews))
+        stringBuilder.append(res.getString(string.stats_overview_forecast_average, oStats.forecastAverageReviews))
         stringBuilder.append("<br>")
-        stringBuilder.append(res.getString(com.ichi2.anki.R.string.stats_overview_forecast_due_tomorrow, oStats.forecastDueTomorrow))
+        stringBuilder.append(res.getString(string.stats_overview_forecast_due_tomorrow, oStats.forecastDueTomorrow))
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/stats/OverviewStatsBuilder.kt:130]: `R.string.xxx` - imported too much`

**Review by david-allison — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/stats/OverviewStatsBuilder.kt (hunk 1 above)
Comment (Jatin887) [reply AnkiDroid/src/main/java/com/ichi2/anki/stats/OverviewStatsBuilder.kt]: changes added

**Review by david-allison — APPROVED** [review]

**Comment by david-allison** [comment]
I guess that works

**Review by mikehardy — APPROVED** [review]
thank you!

**Pull request merged**